midend/parserUnroll.cpp

```diff
@@ -43,6 +45,13 @@ size_t StackVariableHash::operator()(const StackVariable& var) const {
     return Util::Hash::fnv1a(h.data(), sizeof(size_t) * h.size());
 }
 
+const IR::Expression* StackVariable::pathExprToMember(const IR::Expression* expr) {
+    if (expr->is<IR::PathExpression>()) {
```

Comment:
I think that it is a good solution for variable represented by PathExpression. Otherwise, we need to check two possibilities each time: Member or PathExpression.

Reply:
This is a hack, really. It should be at least documented if not avoided.